docroot/modules/custom/mass_content/src/Entity/Bundle/node/NodeBundle.php

```diff
@@ -27,4 +27,28 @@ public function getSearchNoSnippet(): FieldItemListInterface {
     return $this->get('search_nosnippet');
   }
 
+  /**
+   * Gets Organization names slugified.
+   *
+   * @return array
+   *   The array of slugified Org names related to this node.
+   */
+  public function getOrgsSlugified(): array {
+    $result = [];
+    $utilities = \Drupal::service('mass_metatag.utilities');
+    if ($this->bundle() === 'org_page') {
+      $result[] = $utilities->slugify(trim($this->label()));
+    }
+    if ($this->hasField('field_organizations')) {
+      /** @var \Drupal\node\Entity\Node[] $org_pages */
+      $org_pages = $this->field_organizations->referencedEntities();
+      foreach ($org_pages as $org_page) {
+        if ($org_page instanceof OrgPageBundle) {
+          $result[] = $utilities->slugify(trim($org_page->label()));
+        }
+      }
+    }
+    return array_unique($result);
+  }
+
```

Comment:
Thanks for moving this code. In order to match existing methods like [getSearch()](https://github.com/massgov/openmass/blob/8533a197127d71204ee8aee6a66a925f7f7b8844/docroot/modules/custom/mass_fields/src/MassSearchTrait.php#L12), getOrgs() should simply return a FieldItemListInterface(). It should be up to the caller to manipulate further after that. I propose that the slugify call happen  MassUrlGenerator.

If you prefer, I can implement this and then send to you for review.

Reply:
Sounds good.